### PR TITLE
MM-32130 Deleting tasks from an active incident.

### PR DESCRIPTION
**Pull request by crspeller**
#### Summary
Deleting a task from an active incident.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32130

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]
Slick!

File: tests-e2e/cypress/integration/frontstage/incident_rhs_checklist_spec.js [hunk 1]
```diff
@@ -127,5 +127,19 @@ describe('incident rhs checklist', () => {
                 cy.get('.run').eq(1).should('have.text', 'Rerun');
             });
         });
+
+        it.only('delete task', () => {
```
Comment (cpoile) [inline tests-e2e/cypress/integration/frontstage/incident_rhs_checklist_spec.js:131]: Keeping the `.only`?

**Review by cpoile — APPROVED** [review]
Sick!

File: tests-e2e/cypress/integration/frontstage/incident_rhs_checklist_spec.js (hunk 1 above)
Comment (crspeller) [reply tests-e2e/cypress/integration/frontstage/incident_rhs_checklist_spec.js]: Why is it like this... 🤦

**Comment by prapti** [comment]
Overall it's looking great. I have one concern about how the hovering delete button overlaps the timestamp on a task once checked. The timestamp is 'clickable' by design, and the delete button is pretty much hiding it, making it a little awkward to click on the time. As more buttons (favorite, assignee etc. as shown in Figma) get added, it looks like it will completely hide the timestamp. Should this be addressed before this PR is merged? I'm fine if it needs to be discussed in a team meeting, just wanted to bring it up just in case.
Screen Shot 2021-01-26 at 9 15 05 PM

**Comment by crspeller** [comment]
@prapti I noticed that when doing the next ticket in the series here: https://github.com/mattermost/mattermost-plugin-incident-management/pull/439

I think it's fine to merge this PR with that flaw, given that the answer from design around this is just to remove the ability to click on the timestamps: https://community-daily.mattermost.com/core/pl/mnsse7ah73bhxd9wmngtoiu5ph

**Review by prapti — APPROVED** [review]
Got it, thank you for pointing me to the discussion @crspeller!
Approving!

**Pull request merged**